lib/src/cli.rs

```diff
@@ -730,3 +781,24 @@ fn test_parse_generator() {
         Opt::Internals(InternalsOpts::SystemdGenerator { .. })
     ));
 }
+
+#[test]
+fn test_linting() {
+    // linting should only occur in side of a container.
+    match ostree_ext::container_utils::is_ostree_container() {
+        Ok(result) => {
+            if !result {
+                let expected_error_message =
+                    "Not in a ostree container, this command only verifies ostree containers.";
```

Comment:
Hmm well, this will break an IMO theoretically valid use case of doing a build+test inside a bootc container...

Let's switch to having coverage for the kernel and var/run bits which we can easily do in unit tests I think.

Reply:
Noted! and we will attack this shortly

Reply:
> How can I manipulate this data without messing with the real filesystem?

That's one of the awesome things about cap-std, by passing in the rootfs into the lint functions, our unit tests can mock up a rootfs and be *very confident* that the lint checks are not escaping out and interacting with the user's workstation root!

I've updated the code to do that now!